dfastmi/gui/dialog_model.py

```diff
@@ -291,18 +291,17 @@ def _get_condition_configuration(
         measure_files: List,
     ) -> None:
         """Get condition configuration."""
-        num_files = min(len(reference_files), len(measure_files))
-
-        for i, discharge in enumerate(reach.hydro_q[:num_files]):
-            if discharge in reference_files.keys():
+        for i, discharge in enumerate(reach.hydro_q):
```

Comment:
This will crash when the legacy mode is used.

Reply:
Indeed. However, the GUI doesn't support legacy configurations (version 1 files). The batch mode and cli mode do support those, but we don't support them in the gui since the GUI logic would be completely different.

Reply:
Reviewed with Ralph, see #98

Reply:
Discussed with Bert.